env/master.sh

```diff
@@ -1,2 +1,2 @@
-export CBIOPORTAL_URL="${CBIOPORTAL_URL:-https://www.cbioportal.org}"
+export CBIOPORTAL_URL="https://staging.cbioportal.org"
```

Comment:
@dippindots also this should be changed back to www once we we deploy the 7.0 to public